### PR TITLE
Link to GitLab documentation for auth

### PR DESCRIPTION
**Pull request by timrizzi**
Love the docs! I thought it would be helpful to link to GitLab's documentation about how to create an access token.

### CODE REVIEW

File: doc/articles/authentication-for-private-packages.md [hunk 1]
```diff
@@ -223,7 +223,9 @@ composer config [--global] --editor --auth
 > **Note:** For the gitlab authentication to work on private gitlab instances, the
 > [`gitlab-domains`](../06-config.md#gitlab-domains) section should also contain the url.
 
-When creating a gitlab token manually, make sure it has either the `read_api` or `api` scope.
+To create a new access token, go to your [access tokens section on GitLab](https://gitlab.com/-/profile/personal_access_tokens) and create a new token.Read more about it [here](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#creating-a-personal-access-token).
```
Comment (stof) [inline doc/articles/authentication-for-private-packages.md:226]: From an accessibility point of view, linking on the word `here` is not good, as the text of the link itself is not clear.

Comment (stof) [inline doc/articles/authentication-for-private-packages.md:226]: This link to gitlab.com won't work for private gitlab instances.

Comment (denisvmedia) [reply doc/articles/authentication-for-private-packages.md:226]: > new token.Read more about it

A minor issue: missing space after the period.

**Comment by Seldaek** [comment]
Thanks

**Pull request merged**